src/lib/util/assertString.js

```diff
@@ -1,11 +1,4 @@
 export default function assertString(input) {
-  const isString = typeof input === 'string' || input instanceof String;
-
-  if (!isString) {
-    let invalidType = typeof input;
-    if (input === null) invalidType = 'null';
-    else if (invalidType === 'object') invalidType = input.constructor.name;
-
-    throw new TypeError(`Expected a string but received a ${invalidType}`);
-  }
+  if (input === undefined) throw new TypeError(`Expected a string but received a ${input}`);
```

Comment:
should probably check for `null` as well, see https://github.com/validatorjs/validator.js/pull/2372#issuecomment-1936496790

Reply:
I will add null validation and create tests for both cases.